<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PythonConfigDiffWidget</class>
 <widget class="QWidget" name="PythonConfigDiffWidget">
  <property name="geometry">
   <rect>
       <x>0</x>
       <y>0</y>
       <width>490</width>
       <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
     <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
             <widget class="QGroupBox" name="groupBox_rightFile">
                 <property name="styleSheet">
                     <string notr="true">QGroupBox {
                         border: 2px solid rgb(218,218,218);
                         border-radius: 10px;
                         margin-top: 6px;
                         }
                         QGroupBox::title {
                         subcontrol-origin: margin;
                         left: 12px;
                         padding: 0px 3px 0px 3px;

                         }
                     </string>
                 </property>
                 <property name="title">
                     <string notr="true">Load a file to compare...</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_2">
                     <property name="topMargin">
                         <number>12</number>
                     </property>
                     <item row="0" column="0">
                         <widget class="CodeEditor" name="codeEditor_right">
                             <property name="styleSheet">
                                 <string notr="true">selection-color: rgb(0, 0, 0);
                                     selection-background-color: rgb(255, 255, 255);
                                     background-color: none;
                                     border: 2px rgb(0, 0, 0);
                                 </string>
                             </property>
                             <property name="lineWrapMode">
                                 <enum>QPlainTextEdit::NoWrap</enum>
                             </property>
                             <property name="readOnly">
                                 <bool>true</bool>
                             </property>
                             <property name="textInteractionFlags">
                                 <set>Qt::NoTextInteraction</set>
                             </property>
                         </widget>
                     </item>
                 </layout>
             </widget>
         </item>
         <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_leftFile">
                 <property name="styleSheet">
                     <string notr="true">QGroupBox {
                         border: 2px solid rgb(218,218,218);
                         border-radius: 10px;
                         margin-top: 6px;
                         }
                         QGroupBox::title {
                         subcontrol-origin: margin;
                         left: 12px;
                         padding: 0px 3px 0px 3px;

                         }
                     </string>
                 </property>
                 <property name="title">
                     <string notr="true">Load a file to compare...</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_12">
                     <property name="topMargin">
                         <number>12</number>
                     </property>
                     <item row="0" column="0">
                         <widget class="CodeEditor" name="codeEditor_left">
                             <property name="styleSheet">
                                 <string notr="true">QScrollBar:horizontal
                                     {
                                     height: 15px;
                                     margin: 3px 15px 3px 15px;
                                     border: 1px transparent #2A2929;
                                     border-radius: 4px;
                                     background-color: yellow; /* #2A2929; */
                                     }

                                     QScrollBar::handle:horizontal
                                     {
                                     background-color: blue; /* #605F5F; */
                                     min-width: 5px;
                                     border-radius: 4px;
                                     }

                                     QScrollBar::add-line:horizontal
                                     {
                                     margin: 0px 3px 0px 3px;
                                     border-image: url(:/qss_icons/rc/right_arrow_disabled.png);
                                     width: 10px;
                                     height: 10px;
                                     subcontrol-position: right;
                                     subcontrol-origin: margin;
                                     }

                                     QScrollBar::sub-line:horizontal
                                     {
                                     margin: 0px 3px 0px 3px;
                                     border-image: url(:/qss_icons/rc/left_arrow_disabled.png);
                                     height: 10px;
                                     width: 10px;
                                     subcontrol-position: left;
                                     subcontrol-origin: margin;
                                     }

                                     QScrollBar::add-line:horizontal:hover,QScrollBar::add-line:horizontal:on
                                     {
                                     border-image: url(:/qss_icons/rc/right_arrow.png);
                                     height: 10px;
                                     width: 10px;
                                     subcontrol-position: right;
                                     subcontrol-origin: margin;
                                     }


                                     QScrollBar::sub-line:horizontal:hover, QScrollBar::sub-line:horizontal:on
                                     {
                                     border-image: url(:/qss_icons/rc/left_arrow.png);
                                     height: 10px;
                                     width: 10px;
                                     subcontrol-position: left;
                                     subcontrol-origin: margin;
                                     }

                                     QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal
                                     {
                                     background: none;
                                     }


                                     QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
                                     {
                                     background: none;
                                     }

                                     QScrollBar:vertical
                                     {
                                     background-color: #2A2929;
                                     width: 15px;
                                     margin: 15px 3px 15px 3px;
                                     border: 1px transparent #2A2929;
                                     border-radius: 4px;
                                     }

                                     QScrollBar::handle:vertical
                                     {
                                     background-color: red; /* #605F5F; */
                                     min-height: 5px;
                                     border-radius: 4px;
                                     }

                                     QScrollBar::sub-line:vertical
                                     {
                                     margin: 3px 0px 3px 0px;
                                     border-image: url(:/qss_icons/rc/up_arrow_disabled.png);
                                     height: 10px;
                                     width: 10px;
                                     subcontrol-position: top;
                                     subcontrol-origin: margin;
                                     }

                                     QScrollBar::add-line:vertical
                                     {
                                     margin: 3px 0px 3px 0px;
                                     border-image: url(:/qss_icons/rc/down_arrow_disabled.png);
                                     height: 10px;
                                     width: 10px;
                                     subcontrol-position: bottom;
                                     subcontrol-origin: margin;
                                     }

                                     QScrollBar::sub-line:vertical:hover,QScrollBar::sub-line:vertical:on
                                     {
                                     border-image: url(:/qss_icons/rc/up_arrow.png);
                                     height: 10px;
                                     width: 10px;
                                     subcontrol-position: top;
                                     subcontrol-origin: margin;
                                     }

                                     QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on
                                     {
                                     border-image: url(:/qss_icons/rc/down_arrow.png);
                                     height: 10px;
                                     width: 10px;
                                     subcontrol-position: bottom;
                                     subcontrol-origin: margin;
                                     }

                                     QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical
                                     {
                                     background: none;
                                     }

                                     QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
                                     {
                                     background: none;
                                     }
                                 </string>
                             </property>
                             <property name="lineWrapMode">
                                 <enum>QPlainTextEdit::NoWrap</enum>
                             </property>
                             <property name="readOnly">
                                 <bool>true</bool>
                             </property>
                             <property name="textInteractionFlags">
                                 <set>Qt::NoTextInteraction</set>
                             </property>
                         </widget>
      </item>
     </layout>
    </widget>
   </item>
         <item row="1" column="0" colspan="2">
             <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="0">
                     <widget class="QPushButton" name="pushButton_loadLeftFile">
                         <property name="sizePolicy">
                             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                                 <horstretch>0</horstretch>
                                 <verstretch>0</verstretch>
                             </sizepolicy>
                         </property>
                         <property name="text">
                             <string>Load left file</string>
                         </property>
                         <property name="icon">
                             <iconset resource="../../Resources/UI_Icons.qrc">
                                 <normaloff>:/UISymbols/UI_Triangle_Right.svg</normaloff>
                                 :/UISymbols/UI_Triangle_Right.svg
                             </iconset>
                         </property>
                         <property name="iconSize">
                             <size>
                                 <width>18</width>
                                 <height>20</height>
                             </size>
                         </property>
                     </widget>
                 </item>
                 <item row="0" column="1">
                     <widget class="QPushButton" name="pushButton_startDiff">
                         <property name="sizePolicy">
                             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                                 <horstretch>0</horstretch>
                                 <verstretch>0</verstretch>
                             </sizepolicy>
                         </property>
                         <property name="text">
                             <string/>
                         </property>
                         <property name="icon">
                             <iconset resource="../../Resources/UI_Icons.qrc">
                                 <normaloff>:/UISymbols/UI_Diff_Icon.svg</normaloff>:/UISymbols/UI_Diff_Icon.svg
                             </iconset>
                         </property>
                         <property name="iconSize">
                             <size>
                                 <width>40</width>
                                 <height>20</height>
                             </size>
                         </property>
                         <property name="flat">
                             <bool>true</bool>
                         </property>
                     </widget>
                 </item>
                 <item row="0" column="2">
                     <widget class="QPushButton" name="pushButton_loadRightFile">
                         <property name="sizePolicy">
                             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                                 <horstretch>0</horstretch>
                                 <verstretch>0</verstretch>
                             </sizepolicy>
                         </property>
                         <property name="layoutDirection">
                             <enum>Qt::RightToLeft</enum>
                         </property>
                         <property name="text">
                             <string>Load right file</string>
                         </property>
                         <property name="icon">
                             <iconset resource="../../Resources/UI_Icons.qrc">
                                 <normaloff>:/UISymbols/UI_Triangle_Left.svg</normaloff>:/UISymbols/UI_Triangle_Left.svg
                             </iconset>
                         </property>
                         <property name="iconSize">
                             <size>
                                 <width>18</width>
                                 <height>20</height>
                             </size>
                         </property>
                     </widget>
                 </item>
             </layout>
         </item>
     </layout>
 </widget>
    <customwidgets>
        <customwidget>
            <class>CodeEditor</class>
            <extends>QPlainTextEdit</extends>
            <header location="global">codeeditor.h</header>
        </customwidget>
    </customwidgets>
    <resources>
        <include location="../../Resources/UI_Icons.qrc"/>
    </resources>
    <connections/>
</ui>
